.eslintrc.json

```diff
@@ -18,7 +18,6 @@
   "rules": {
     "prettier/prettier": [
       "error", {
-        "printWidth": 80,
```

Comment:
Why? You love reading long lines?

Reply:
Actually, i was getting an error about eslint then i have checked from stackoverflow and there is a solution "you need to delete printWidth argument, it will work". That's why i did it .

Reply:
What kind of error was it? 
Was it Eslint complaining about lines in your code being longer than 80 characters or what was it?